services/web/server/tests/unit/with_dbs/03/folders/test_folders.py

```diff
@@ -336,3 +336,101 @@ async def test_project_listing_inside_of_private_folder(
         data, _ = await assert_status(resp, status.HTTP_200_OK)
         assert len(data) == 1
         assert data[0]["uuid"] == user_project["uuid"]
+
+
+@pytest.mark.parametrize("user_role,expected", [(UserRole.USER, status.HTTP_200_OK)])
```

Comment:
this feels a copy&paste since you do not even use `expected`. 
TIP:  If all examples here are with the same `user_role`, just define a fixture on this file as
```python

@pytest.fixture
def user_role() -> UserRole:
     # NOTE: affects `logged_user` fixture
     return UserRole.USER

async def test_folders_deletion(
    client: TestClient,
    logged_user: UserInfoDict,
    user_project: ProjectDict,
    mock_catalog_api_get_services_for_user_in_product: MockerFixture,
):
    assert logged_user['role'] == UserRole.USER
    # ... 
```